feat: readme enhanced (#117)
Stadt-Geschichte-Basel/stadtgeschichtebasel.ch: README.md

# stadtgeschichtebasel.ch

> The open-source code of the digital portal of Stadt.Geschichte.Basel. [![GitHub issues](https://img.shields.io/github/issues/Stadt-Geschichte-Basel/stadtgeschichtebasel.ch.svg)](https://github.com/Stadt-Geschichte-Basel/stadtgeschichtebasel.ch/issues)

Reviewer: ```suggestion
The open-source code of the digital portal of Stadt.Geschichte.Basel.
```

This project is the open-source code of the digital portal of Stadt.Geschichte.Basel. It is a static website built with [SvelteKit](https://kit.svelte.dev/) and hosted on [Cloudflare Pages](https://pages.cloudflare.com/). It uses [hypotheses.org](https://sgb.hypotheses.org/) as CMS and consumes data from [Agendabasel](https://agendabasel.ch/) for the agenda.

The digital platform for research data can be found at [forschung.stadtgeschichtebasel.ch](https://forschung.stadtgeschichtebasel.ch/). The source code for the digital platform can be found [here](https://github.com/Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch).

## Key Features

- **Fast Static Website**: The website is built with SvelteKit, providing a fast and responsive user experience.
- **Interactive Features**: The website includes an interactive map and agenda.

> - **Accessibility-Focused Design**: Ensuring inclusivity for all users by complying with WCAG standards and observing neurodiversity design guidelines. More at our [accessibility statement](https://stadtgeschichtebasel.ch/barrierefreiheit/).

Reviewer: ```suggestion
- **Accessibility-Focused Design**: Ensuring inclusivity for all users by complying with WCAG standards and observing neurodiversity design guidelines. More at our [accessibility statement](https://stadtgeschichtebasel.ch/barrierefreiheitserklaerung/).
```

[![GitHub stars](https://img.shields.io/github/stars/Stadt-Geschichte-Basel/stadtgeschichtebasel.ch.svg)](https://github.com/Stadt-Geschichte-Basel/stadtgeschichtebasel.ch/stargazers)
[![GitHub license](https://img.shields.io/github/license/Stadt-Geschichte-Basel/stadtgeschichtebasel.ch.svg)](https://github.com/Stadt-Geschichte-Basel/stadtgeschichtebasel.ch/blob/main/LICENSE.md)

## Overview

Welcome to [Stadt.Geschichte.Basel](https://stadtgeschichtebasel.ch/), a comprehensive digital and print project aimed at presenting the multifaceted history of Basel from its earliest beginnings to the present. Visit [Stadt.Geschichte.Basel](https://stadtgeschichtebasel.ch) to see how our digital portal is bringing Basel's history to life. This live site showcases the practical application of our open-source code, offering an interactive experience of Basel's rich heritage.

> [Stadt.Geschichte.Basel](https://stadtgeschichtebasel.ch/) seeks to bridge research gaps and present historical findings in accessible formats. Our project encompasses an extensive nine-volume book series, an overview volume, a digital portal, and a digital platform for research data. These resources are aimed at scholars and the public, making historical research and insights more accessible than ever.

Reviewer: ```suggestion
[Stadt.Geschichte.Basel](https://stadtgeschichtebasel.ch/) seeks to bridge research gaps and present historical findings in accessible formats. Our project encompasses an [extensive nine-volume book series](https://www.merianverlag.ch/buecher/stadt.geschichte.basel.html), an overview volume, a digital portal, and a digital platform for research data. These resources are aimed at scholars and the public, making historical research and insights more accessible than ever.
```

> This project is the open source code of the digital portal of Stadt.Geschichte.Basel. It is a static website built with [SvelteKit](https://kit.svelte.dev/). The website is hosted on [Cloudflare Pages](https://pages.cloudflare.com/). It uses [Hypothese.org](https://sgb.hypotheses.org/) as CMS and consumes data from [Agendabasel](https://agendabasel.ch/) for the agenda.

Reviewer: ```suggestion
This project is the open source code of the digital portal of Stadt.Geschichte.Basel. It is a static website built with [SvelteKit](https://kit.svelte.dev/). The website is hosted on [Cloudflare Pages](https://pages.cloudflare.com/). It uses [hypotheses.org](https://sgb.hypotheses.org/) as CMS and consumes data from [Agendabasel](https://agendabasel.ch/) for the agenda.
```

The digital platform for research data can be found at [forschung.stadtgeschichtebasel.ch](https://forschung.stadtgeschichtebasel.ch/). The source code for the digital platform can be found [here](https://github.com/Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch).

## Key Features

- **Fast Static Website**: The website is built with SvelteKit, providing a fast and responsive user experience.
- **Interactive Features**: The website includes an interactive map, and agenda.

> - **Accessibility-Focused Design**: Compliant with WCAG standards, ensuring inclusivity for all users. More at our [accessibility statement](https://stadtgeschichtebasel.ch/barrierefreiheit/).

Reviewer: ```suggestion
- **Accessibility-Focused Design**: Ensuring inclusivity for all users, by complying with WCAG standards and observing neurodiversity design guidelines. More at our [accessibility statement](https://stadtgeschichtebasel.ch/barrierefreiheit/).
```

pnpm install
```

> Run the prebuild script to download the pages, posts, and assets from [Hyphotheses.org](https://sgb.hypotheses.org/).

Reviewer: ```suggestion
Run the prebuild script to download the pages, posts, and assets from [hypotheses.org](https://sgb.hypotheses.org/).
```
